Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated_v3.x.nuspec

```diff
@@ -27,5 +27,6 @@
     <file src="..\..\..\Functions.Templates\Templates\CosmosDBTrigger-CSharp-Isolated\**" target="content\CosmosDBTrigger-CSharp" />
     <file src="..\..\..\Functions.Templates\Templates\SignalRConnectionInfoHttpTrigger-CSharp-Isolated\**" target="content\SignalRConnectionInfoHttpTrigger-CSharp" />
     <file src="..\..\..\Functions.Templates\Templates\RabbitMQTrigger-CSharp-Isolated\**" target="content\RabbitMQTrigger-CSharp" />
+    <file src="..\..\..\Functions.Templates\Templates\EventGridBlobTrigger-CSharp-Isolated\**" target="content\EventGridBlobTrigger-CSharp" />
```

Comment:
This package is not used anymore, Only need to update 
Build/PackageFiles/Dotnet_precompiled/ItemTemplates-Isolated_v4.x.nuspec

Reply:
Removed